### DISCUSSION TITLE
Setting up SDL2 on Windows

### DISCUSSION THREAD

**Discussion by ThePhantomMask** [SagaPDev/Godot-SDL-Gyro#6] [Q&A]
I cannot continue with the compilation of the addon because it does not open SDL2.lib, I don't know what to do

**Comment by SagaPDev** [comment]
i will take a look at your fork

**Comment by SagaPDev** [comment]
i think this is my fault the build instructions had a typo, try again with `scons use_mingw=yes`

**Reply by ThePhantomMask** [reply]
If I do it via a Linux virtual machine and then compile there, it would also work on Windows, right?

**Reply by SagaPDev** [reply]
In theory it should work, but it doesn't, i have to boot windows to compile the windows version, i tested your fork on there and it compiles successfully, there shouldn't be a need for for you to use VM

**Reply by ThePhantomMask** [reply]
I don't know why but compilation doesn't work for me

**Reply by SagaPDev** [reply]
could you show me the contents of your libs folder? i cloned your fork everything else should be the same

**Reply by ThePhantomMask** [reply]
![Screenshot (204)](https://github.com/SagaPDev/Godot-SDL-Gyro/assets/74540016/0c12b831-10c5-4311-bd62-a66eb426f9e5)

**Reply by ThePhantomMask** [reply]
Anyway... I tried this scons use_mingw=yes but it gives me an error.
Another error: link /nologo /WX /OPT:REF /OPT:NOICF /dll /out:Godot_Gamepad_SDLTest\addons\godot-sdl-gyro\sdlgyro.windows.template_debug.x86_64.dll /implib:Godot_Gamepad_SDLTest\addons\godot-sdl-gyro\sdlgyro.windows.template_debug.x86_64.lib godot-cpp\bin\libgodot-cpp.windows.template_debug.x86_64.lib SDL2.lib SDL2.lib SDL2main.lib src\register_types.windows.template_debug.x86_64.obj src\sdlgyro.windows.template_debug.x86_64.obj
LINK : fatal error LNK1181: it is not possible to open the 'SDL2.lib'
scons: *** [Godot_Gamepad_SDLTest\addons\godot-sdl-gyro\sdlgyro.windows.template_debug.x86_64.dll] Error 1181
scons: building terminated because of errors.

**Reply by SagaPDev** [reply]
yeah seems fine, this is very estrange... what errors `scons use_mingw=yes` give you? (a picture of just the last error messages is fine)

**Reply by ThePhantomMask** [reply]
scons: *** [godot-cpp\src\godot.windows.template_debug.x86_64.o] Cannot find the specified file
scons: *** [godot-cpp\src\classes\editor_plugin_registration.windows.template_debug.x86_64.o] Cannot find the specified file
scons: *** [godot-cpp\src\classes\low_level.windows.template_debug.x86_64.o] Cannot find the specified file

**Reply by SagaPDev** [reply]
try cleaning the build files with `scons -c` and then try again with `scons use_mingw=yes` i will make this an issue and investigate

**Reply by ThePhantomMask** [reply]
Gives me error: scons: *** [godot-cpp\src\core\memory.windows.template_debug.x86_64.o] Cannot find the specified file
